components/Order/model_interfaces.rst

```diff
@@ -0,0 +1,84 @@
+Model interfaces
+================
+
+.. _component_order_model_adjustment-interface:
+
+AdjustmentInterface
+-------------------
+This interface provides you with basic management of an adjustment's properties.
+
+.. note::
+    This interface extends the :ref:`component_resource_model_timestampable-interface` and the :ref:`component_originator_model_origin-aware-interface` |br|
+    For more detailed information about the interface go to `Sylius API's AdjustmentInterface`_.
+.. _Sylius API's AdjustmentInterface: http://api.sylius.org/Sylius/Component/Order/Model/AdjustmentInterface.html
+
+.. _component_order_model_adjustable-interface:
+
+AdjustableInterface
+-------------------
+This interface provides basic tools for adjustment management.
+Use this interface to any model that you want to be adjustable.
+For example following models implement this interface
+    * :ref:`component_resource_model_order`
+    * :ref:`component_resource_model_order-item`
+
+.. note::
+    For more detailed information about the interface go to `Sylius API's AdjustableInterface`_.
+.. _Sylius API's AdjustableInterface: http://api.sylius.org/Sylius/Component/Order/Model/AdjustableInterface.html
+
+.. _component_order_model_comment-interface:
+
+CommentInterface
+----------------
+This interface provides you with basic management of a comment's properties.
+
+.. note::
+    This interface extends the :ref:`sylius_component_resource_model_timestampable-interface` |br|
+    For more detailed information about the interface go to `Sylius API's CommentInterface`_.
+.. _Sylius API's CommentInterface: http://api.sylius.org/Sylius/Component/Order/Model/CommentInterface.html
+
+.. _component_order_model_comment-aware-interface:
+
+CommentAwareInterface
+---------------------
+This interface provides basic tools for comments management.
+If you want to have comments in your model just implement this interface.
+
+.. note::
+    For more detailed information about the interface go to `Sylius API's CommentAwareInterface`_.
+.. _Sylius API's CommentAwareInterface: http://api.sylius.org/Sylius/Component/Order/Model/CommentAwareInterface.html
+
+.. _component_order_model_order-interface:
+
+OrderInterface
+--------------
+This interface provides you with basic management of an order's properties.
+
+.. note::
+    This interface extends the :ref:`component_resource_model_timestampable-interface`, the :ref:`component_resource_model_timestampable-interface`, |br|
+    the :ref:`component_order_model_adjustable-interface`, the :ref:`component_order_model_comment-aware-interface`, |br|
+    the :ref:`component_resource_model_soft-deletable-interface` and the :ref:`component_sequence_model_sequence-subject-interface`. |br|
```

Comment:
A little too many `|br|`. Think one at the end would suffice, also think You can drop the `the` when referring more than 2 interfaces.